docs/vgclog.md

```diff
@@ -0,0 +1,207 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Verbose Garbage Collection Logs 
+
+[Garbage collection](https://www.eclipse.org/openj9/docs/gc/)(gc) reclaims unused memory in the heap for reuse. During cleanup of the heap, the verbose garbage collection (vgc) logs capture information about the different garbage collection operations that are involved in the garbage collection cycles. Gc operations include GC initialization, stop-the-world processing, finalization, reference processing, and allocation failures. 
+
+A garbage collection event is triggered in the following situations: 
+
+- The JVM calls system.gc() to run a global STW gc. 
+
+- A memory allocation failure. Allocation failures are triggered when a request to allocate an object fails due to a full region of the heap. 
+
+- Decrease in available memory to a value that triggers a concurrent cycle to clean the heap or a region of the heap. 
+
+- Concurrent events that are part of a concurrent cycle are complete, which triggers an STW event that completes either an incremental or final part of the concurrent.
+
+Verbose garbage collection (vgc) logs contain information on garbage collection operations to assist with the following actions:  
+
+- Tuning the performance of the garbage collectors 
+
+- Troubleshooting gc operations and policies. For example, analyzing long pauses, or determining how free memory is divided in the heap before and after a gc cycle. 
+
+When verbose garbage collection (vgc) logs are enabled, the logs are collected as soon as [garbage collection](gc.md) is initialized. 
```

Comment:
"...the logs are collected as soon as GC is initialized"? This might imply that they are complete, I think?

Maybe  "...the logs start to generate as soon as..."

Reply:
Changed to *Verbose GC logs, when enabled, begin capturing information as soon as garbage collection is initialized.*